lib/index.js

```diff
@@ -13,12 +15,20 @@ const linterUiDefault = {
       atom.packages.getLoadedPackage('linter-ui-default').metadata['package-deps'].push('busy-signal')
     }
 
-    if (!atom.inSpecMode()) {
-      // eslint-disable-next-line global-require
-      require('atom-package-deps').install('linter-ui-default', true)
+    let callbackID
+    const installLinterUIDefaultDeps = () => {
```

Comment:
I guess we have to pick between the two: more verbose profiling vs cleaner code
I would personally prefer cleaner code because we can always do specific modifications when profiling, and profiler also shows the file name the function is from so we already have an idea of where it's coming from